code/modules/cargo/bounties/security.dm

```diff
@@ -26,25 +26,25 @@
 /datum/bounty/item/security/pinpointer
 	name = "Nuclear Pinpointer"
 	description = "There's a teeny-tiny itty-bitty chance CentCom may have lost a nuke disk. Can the station spare a pinpointer to help out?"
-	reward = 1500
+	reward = 5000
 	wanted_types = list(/obj/item/pinpointer/nuke)
 
 /datum/bounty/item/security/captains_spare
 	name = "Captain's Spare"
 	description = "Captain Bart of Station 12 has forgotten his ID! Ship him your station's spare, would you?"
-	reward = 1500
+	reward = 5000
```

Comment:
- (8) these items may be sold during war ops or other 1 vs all antag types so the price should not be as high as when there arent and not as low as when there are